metricbeat/metricbeat.reference.yml

```diff
@@ -317,6 +317,14 @@ metricbeat.modules:
   metricsets:
     - event
 
+#--------------------------------- kvm Module --------------------------------
+- module: kvm
+  metricsets: ["memory"]
```

Comment:
`memory` still here, you may need to run `make update` again.

Reply:
Done :)